md-dao/src/main/java/com/wipro/ats/bdre/md/dao/AnalyticsAppsDAO.java

```diff
@@ -0,0 +1,112 @@
+package com.wipro.ats.bdre.md.dao;
+
+import com.wipro.ats.bdre.exception.MetadataException;
+import com.wipro.ats.bdre.md.beans.table.AnalyticsApps;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wipro.ats.bdre.md.beans.table.AnalyticsApps'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AUselessImportCheck)